autoware_plugin/include/autoware_plugin.h

```diff
@@ -57,14 +61,34 @@ namespace autoware_plugin
         std::shared_ptr<ros::CARMANodeHandle> nh_, pnh_;
 
         // publisher for CARMA trajectory
-        ros::Publisher  trajectory_pub_;
+        // ros::Publisher  trajectory_pub_;
         ros::Publisher autoware_plugin_discovery_pub_;
 
         // subscriber for Autoware waypoints
         ros::Subscriber waypoints_sub_;
         ros::Subscriber pose_sub_;
         ros::Subscriber twist_sub_;
 
+
+
+        // ros service servers
+        ros::ServiceServer trajectory_srv_;
+        ros::ServiceServer maneuver_srv_;
+
+        // service callbacks
```

Comment:
Provide a little bit more detail in this comment. "service callbacks for carma planning" something like that

Reply:
details added